code/client/repoclean

```diff
@@ -454,6 +462,11 @@ def main():
 
     if not options.plugin:
         options.plugin = 'FileRepo'
+    
+    if not options.auto:
+        options.auto = False
+    else:
+        options.auto = True
```

Comment:
This seems redundant, especially if you add the default=False to the option definition.